docs/development/java/install-java-on-ubuntu-18-04/index.md

```diff
@@ -31,36 +31,51 @@ This guide demonstrates installing Oracle's Java Standard Edition Development Ki
 
 - You will need root access to your Linode, or a [limited user account](/docs/security/securing-your-server/#add-a-limited-user-account) with `sudo` privileges.
 
-- Update your system:
+- Connect to your Linode using a remote desktop sharing system such as a VNC. For more information, see [Install VNC on Ubuntu 18.04](/docs/applications/remote-desktop/install-vnc-on-ubuntu-18-04/) guide.
 
-        sudo apt update && sudo apt upgrade
+- Create an Oracle account and download Oracle JDK 11 .tar.gz [archive] (https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html). Ensure that the Oracle JDK version is the same as the oracle-java11-installer-local [package version] (https://launchpad.net/~linuxuprising/+archive/ubuntu/java/+packages).
```

Comment:
[Login link for Oracle account](https://login.oracle.com/oaam_server/login.do) should be added too.